scripts/release/utils/jira_helper.py

```diff
@@ -163,6 +173,7 @@ def get_auth_header(username, password):
         'Authorization': auth_helper.encode_to_auth_header(username, password)
     }
 
+
```

Comment:
It's in the Python's style guide, to separate functions with 2 blank lines. I've missed this last time. https://www.python.org/dev/peps/pep-0008/#blank-lines